### ISSUE TITLE
Exit code if boundary assignment is not succesfull

### ISSUE THREAD

**Issue by JensMadsen** [boutproject/BOUT-dev#29]
If in BOUT.inp by mistake i write

bndry_all = dirchlet_oO

instead of 

bndry_all = dirchlet_o0

BOUT++ defautls to std dirichlet boundary operator

**Comment by dschwoerer** [comment]
Is this fixed by PR #126?

**Comment by loeiten** [comment]
Indeed it is. If for example `bndry_all = dirchlet_04` is typed instead of `bndry_all = dirchlet_o4`, BOUT++ now throws the error

```
Error encountered
====== Back trace ======
 -> Adding 3D field: Solver::add(T)
 -> Initialising physics module
  Boundary setting is neither an operation nor modifier: dirichlet_04
```

instead of falling back to `dirichlet`
